### PR TITLE
Remove traces of renderdb

### PR DESCRIPTION
**Pull request by sumarlidason**
refs #1255

* remove traces of mapnik access

### CODE REVIEW

**Comment by sumarlidason** [comment]
pairs w/ 

https://github.com/ngageoint/hootenanny/issues/2700
https://github.com/ngageoint/hootenanny/pull/2725

File: js/id/renderer/background.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ iD.Background = function(context) {
     //Added to remove the Locator Overlay layer on layers that have their own labels on
     //the initial launch of the map
     function checkLocatorNeed(locatorNeed){
-        if (locatorNeed === 'MAPNIK' || locatorNeed === 'USGS Topographic Maps'){
```
Comment (brianhatchl) [inline js/id/renderer/background.js]: Revert this.  Unrelated to our node-mapnik renderdb removal.

File: js/id/ui/background.js [hunk 2]
```diff
@@ -81,7 +81,7 @@ iD.ui.Background = function(context) {
                     .filter(function(d) { return d.overlay;});
 
                 var layerUsed = d.imageryUsed();
-                if (layerUsed === 'MAPNIK' || layerUsed === 'USGS Topographic Maps'){
```
Comment (brianhatchl) [inline js/id/ui/background.js]: Revert this.  Unrelated to our node-mapnik renderdb removal.

File: data/imagery.json [hunk 3]
```diff
@@ -30836,20 +30836,6 @@
...
-        "template": "http://{switch:a,b,c}.tile.openstreetmap.org/{zoom}/{x}/{y}.png",
-        "scaleExtent": [
-            0,
-            19
-        ],
-        "terms_url": "http://openstreetmap.org/",
-        "terms_text": "© OpenStreetMap contributors, CC-BY-SA",
-        "id": "MAPNIK",
-        "default": true
-    },
```
Comment (brianhatchl) [inline data/imagery.json]: Revert this.  Unrelated to our node-mapnik renderdb removal.

**Pull request merged**